static/app/components/events/eventStatisticalDetector/eventDifferentialFlamegraph.tsx

```diff
@@ -66,7 +72,37 @@ export function EventDifferenialFlamegraph(props: EventDifferenialFlamegraphProp
   return (
     <div>
       <h3>Differential Flamegraph</h3>
-
+      <div>
```

Comment:
Temporary here for easily visualizing different txns, will be removed in following PR's